packages/transformers/html/src/inline.js

```diff
@@ -49,12 +52,37 @@ export default function extractInlineAssets(
           type = 'js';
         }
 
+        if (!node.attrs) {
+          node.attrs = {};
+        }
+
+        // allow a script/style tag to declare its key
+        if (node.attrs['data-parcel-key']) {
+          parcelKey = node.attrs['data-parcel-key'];
+        }
+
+        // Inform packager to remove type, since CSS and JS are the defaults.
+        // Unless it's application/ld+json
+        if (
+          node.attrs &&
+          (node.tag === 'style' ||
+            (node.attrs.type && SCRIPT_TYPES[node.attrs.type] === 'js'))
+        ) {
+          node.attrs['parcel-should-delete-type'] = 'true';
```

Comment:
Do we need the type later? Or could you just delete it here instead?

Reply:
yea, I put the delete here for clarity.